content/documentation/features/index.adoc

```diff
@@ -22,25 +22,25 @@ menu:
 
 == Overview
 
-Kiali helps you define, validate, and observe your Istio service mesh. Kiali provides answers to the questions: What are the microservices in my service mesh, and how are they connected?
+Kiali helps you define, validate, and observe the connections of your microservices in an Istio service mesh.
```

Comment:
This may be too narrow as-is, it's not just the connections, but the microservices and the connections. maybe:

> Kiali helps you define, validate, and observe your Istio service mesh microservices and how they connect.